src/itdelatrisu/opsu/Options.java

```diff
@@ -223,10 +225,13 @@ public void click(GameContainer container) {
 
 			@Override
 			public void read(String s) {
+				String str = "?";
 				try {
-					Resolution res = Resolution.valueOf(String.format("RES_%s", s.replace('x', '_')));
-					resolution = res;
-				} catch (IllegalArgumentException e) {}
+					str = String.format("RES_%s", s.replace('x', '_'));
+					resolution = Resolution.valueOf(str);
+				} catch (IllegalArgumentException e) {
+					ErrorHandler.error(String.format("Invalid resolution in config file: %s", str), e, false);
```

Comment:
All other errors in reading the configuration file are ignored, which is why I purposefully didn't throw an error for this, either.